### PR TITLE
changed slack api calls in app.coffee

### PR DESCRIPTION
**Pull request by ayanchoudhary**
@vrongmeal pls review once to see if I am using the correct calls

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "sanitize-html": "^1.14.1",
     "showdown": "^1.7.6",
     "showdown-htmlescape": "^0.1.9",
+    "slack": "^11.0.2",
     "slack-utils": "^0.3.2"
```
Comment (captn3m0) [inline package.json]: remove slack-utils as well

**Comment by vrongmeal** [comment]
@AyanChoudhary test your changes and then we'll proceed. I think the first step is to authorise using the new package. And then we can move on to changing methods.

**Comment by vrongmeal** [comment]
Did you test these changes on your local?

**Comment by captn3m0** [comment]
@AyanChoudhary @vrongmeal any update on this?

**Comment by AyanChoudhary** [comment]
> @AyanChoudhary @vrongmeal any update on this?

@captn3m0 we were thinking of rewriting this code a-new. So we need your input as to whether it will be feasible or that we should make changes in the existing one.

Our primary reasons for rewriting were to remove the circular dependencies in cases such as `announcementHandler = require('./announcements.coffee')(app.io, slack)
userInfoHandler = require('./userinfo.coffee')(app.io, slack)
`
So we wanted to know your views on this.

**Comment by vrongmeal** [comment]
Hey @captn3m0 . I did not get a chance to look into the problem that @AyanChoudhary faced (was occupied due to internship tests and interviews). As far as I know, it wouldn't be hard to rewrite but not sure if we should. Ayan could not resolve the issues by making just a few changes and replacing the package.

**Comment by captn3m0** [comment]
The other alternative is to upgrade the ws dependency on the slack-utils package.

**Comment by vrongmeal** [comment]
@AyanChoudhary can you try upgrading `ws` in https://github.com/captn3m0/slack-utils ?

**Comment by AyanChoudhary** [comment]
> The other alternative is to upgrade the ws dependency on the slack-utils package.

@captn3m0 i tried updating the ws dependency but i don't have the right s to create a pr on slack-utils repo

**Comment by captn3m0** [comment]
@AyanChoudhary You can always fork and file a PR.

Gave you access.

**Comment by c0dzilla** [comment]
Refers #71

**Comment by ayanchoudhary** [comment]
fixed in #80

**Pull request closed without merging**